ftml/conf/blocks.schema.toml

```diff
@@ -0,0 +1,92 @@
+# Schema for blocks.toml
+#
+# This describes what properties bloks can have, as mentioned in docs/Blocks.md
+
+[block-name]
+
+# Whether this block is deprecated or not.
+# Deprecated blocks should be clearly marked as such, and
+# should not be actively recommended to users.
+#
+# If this is set, then the deprecation-message in the corresponding
+# language file should be populated.
+#
+# type: bool, optional
+deprecated = false
+
+# What aliases (if any) this block accepts, in addition to its name.
+# Case-insensitive.
+#
+# type: string[], optional
+aliases = ["a", "anchor"]
+
+# Whether this block accepts the star (*) modifier.
+#
+# type: bool, optional
+accepts-star = false
+
+# Whether this block accepts the score (_) modifier.
+#
+# type: bool, optional
+accepts-score = true
+
+# Whether this block accepts newline delimition for its sections.
+#
+# type: bool, optional
+accepts-newlines = true
+
+# What kind of head this block expects.
+# See the document "Blocks.md" for more information.
+#
+# type: string, enum
+# values: "none", "value", "map", "value+map"
+head = "map"
+
+# What kind of body this block expects, if any.
+#
+# type: string, enum
+# values: "none", "raw", "elements", "other"
+body = "elements"
+
+# Which arguments (if any) this block accepts.
+# Does not include HTML attributes, see below.
+#
+# Only applies to blocks that have head = "map" or "value+map".
+#
+# type: string[], optional
+arguments = ["hideLocation", "show", "hide"]
```

Comment:
I feel like arguments are going to need more detail than this - e.g. whether or not they're a boolean attribute, enumerated, enumerated + any value, etc.

Reply:
Okay, I'll add that tomorrow.